app/views/post_peep.erb

```diff
@@ -0,0 +1,8 @@
+<div id='post_peep'>
+  <form action='/peep/new' method='post'>
+    <h3> Post a peep! </h3>
+    <textarea name='post_peep' placeholder='What are you thinking?'></textarea>
```

Comment:
Style this with CSS if you want it bigger -> `input type="text"` will default to a `VARCHAR` datatype in SQL, which will be quicker than a big text thing